docs/build-framework/switches/diagnostics.md

```
@@ -0,0 +1,33 @@
+---
+seo_title: "Armbian build output & diagnostics switches"
+description: "Diagnostics for Armbian builds: SHARE_LOG to upload full build logs, SHOW_LOG to stream command output live, and SHOW_DEBUG for verbose messages."
+---
+
+# Output & diagnostics
+
+Logs and diagnostic output for a build.
+
+#### SHARE_LOG
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)